### PR TITLE
ddl: fix column value exist backfill bug.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**
Consider scenario(t table(c1 int, c2 int), c2 is the new added column):
1 A is in public state, add a row(1,3);
2 B is in reorg state, check c2 value exists(here is 3), then it will do nothing.

Current implementation doesn't check new added column value exists, so it is a bug here.
Now it is not easy to build a test case to cover it, i will add it in integration testing later.

### CODE REVIEW

**Comment by qiuyesuifeng** [comment]
@siddontang  PTAL

File: ddl/column.go [hunk 1]
```diff
@@ -366,10 +366,13 @@ func (d *ddl) backfillColumnData(t table.Table, columnInfo *model.ColumnInfo, ha
 			}
 
 			backfillKey := t.RecordKey(handle, &column.Col{ColumnInfo: *columnInfo})
-			_, err = txn.Get(backfillKey)
+			backfillValue, err := txn.Get(backfillKey)
 			if err != nil && !kv.IsErrNotFound(err) {
 				return errors.Trace(err)
 			}
+			if backfillValue != nil {
```
Comment (siddontang) [inline ddl/column.go]: need comment here?

File: ddl/column.go [hunk 2]
```diff
@@ -304,6 +304,12 @@ func (d *ddl) onDropColumn(t *meta.Meta, job *model.Job) error {
 	}
 }
 
+// How to backfill column data in reorganization state?
+//  1, Generate a snapshot with special version.
```
Comment (ngaut) [inline ddl/column.go]: s/,/./g

**Comment by qiuyesuifeng** [comment]
PTAL

**Comment by siddontang** [comment]
LGTM

**Pull request merged**